rules/windows/process_creation/proc_creation_win_uac_bypass_hijacking_firwall_snap_in.yml

```diff
@@ -0,0 +1,24 @@
+title: UAC Bypass via Windows Firewall Snap-In Hijack
+id: e52cb31c-10ed-4aea-bcb7-593c9f4a315b
+status: experimental
+description: Detects attempts to bypass User Account Control (UAC) by hijacking the Microsoft Management Console (MMC) Windows Firewall snap-in
+references:
+    - https://www.elastic.co/guide/en/security/current/uac-bypass-via-windows-firewall-snap-in-hijack.html#uac-bypass-via-windows-firewall-snap-in-hijack
+author: Tim Rauch
+date: 2022/09/27
+logsource:
+    category: process_creation
+    product: windows
+tags:
+    - attack.privilege_escalation
+    - attack.t1548
+detection:
+    selection:
+        ParentImage|endswith: '\mmc.exe'
+        ParentCommandLine|contains: 'WF.msc'
+    selection_not:
```

Comment:
Using `filter` instead of "selection_not" is a best practice for the comprehension